.github/workflows/update-docs-webhook.yaml

```diff
@@ -0,0 +1,21 @@
+name: Update docs webhook
+on:
+  push:
+    branches:
+      - master
+      - branch/v*
+  workflow_dispatch:
+
+jobs:
+  update-webhook:
+    name: Update docs webhook
+    runs-on: ubuntu-latest
+    environment: update-docs
+    steps:
+      - name: Call deployment webhook
+        env:
+          WEBHOOK_URL: ${{ secrets.DOCS_DEPLOY_HOOK }}
+        run: |
+          if curl --silent --fail --show-error "$WEBHOOK_URL" > /dev/null; then
```

Comment:
I talked with earlier today @webvictim and this is actually correct. [Based on the docs](https://vercel.com/docs/deployments/deploy-hooks) it looks like the payload content is not actually used.

This does seem a little strange to me as well, but I am able to curl the `WEBHOOK_URL` locally and it does appear to trigger a deployment each time.

Reply:
What does it deploy then? If it just updates the master docs then there's no point backporting this or having the `branch/v*` trigger. I assumed the payload must be relevant for it to work on `branch/v*` but if that's not the case perhaps we don't even need to trigger on those branches?

Reply:
Thinking a little more, we probably do not want to deploy on branch/v* as we should only do that when we cut a release, otherwise the docs for the release branches get ahead of the actual release. This is fine on master, and possibly a pre-release branch (like v15 currently), but we should not be doing any sort of deployment on a push to branch/v*

Reply:
> What does it deploy then?

Quite frankly, no idea. @webvictim, do you know what gets deployed?

> we probably do not want to deploy on branch/v* as we should only do that when we cut a release, otherwise the docs for the release branches get ahead of the actual release

That's fine with me, but given that our team does not handle vercel/docs deployments (outside of this single webhook call) I think some other team would need to update this in a separate PR.